src/pages/guides/configuration/index.md

```diff
@@ -131,6 +131,28 @@ runtimeManifest
 > Note that the above example also demonstrates the 'include' field of an action.  In some cases you may want to have a file deployed with your action code, and available to your code when it runs.
 The example will copy all .txt files from the `myfilestoinclude/` directory and place it in a new dir `text/` that is available via `fs.readFile('text/somefile.txt', 'utf8', callback);` when your action is invoked.
 
+##### Annotations 
+
+Runtime actions can be decorated with annotations to enhance or modify action behavior. 
+
+```yaml
+runtimeManifest:
+   packages:
+     myapp:
+       license: Apache-2.0
+       actions:
+         generic:
+           annotations:
+             require-adobe-auth: true
+             disable-download: true  
+```
+
+
+In addition to the annotations provided out of the box by Apache Openwhisk (See [here](https://github.com/apache/openwhisk/blob/master/docs/annotations.md#annotations-on-openwhisk-assets)), there are a few annotations specific to I/O Runtime: 
```

Comment:
I wouldn't add "Apache Openwhisk" here, instead just use Runtime in order to avoid confusion. The same documentation is also available here - https://github.com/adobe-apiplatform/incubator-openwhisk/blob/master/docs/annotations.md